woodwork/tests/requirement_files/latest_core_dependencies.txt

```diff
@@ -1,4 +1,4 @@
 numpy==1.22.3
-pandas==1.4.1
+pandas==1.4.2
```

Comment:
```suggestion
pandas==1.4.1
```

Reply:
You don't want to increase this, until Woodwork supports that pandas version

Reply:
Oh weird, I thought I had set this to 1.4.1. Will change!